test/java/seedu/addressbook/data/person/NameTest.java

```diff
@@ -8,7 +8,10 @@
 
     @Test
     public void testSimilar_randomTwoNamesIdentical_shouldReturnTrue() {
+        Name a = new Name("XuChen");
+        Name b = new Name("XuChen");
         
+        assertTrue(a.isSimilar(b));
```

Comment:
In TDD, you should start with all fail test cases before start writing working test cases.